src/Components/Artwork/Details/PartnerOfferedPrice.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Flex, Text } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { SaleMessage } from "./Details"
+import { PartnerOfferedPriceQuery } from "__generated__/PartnerOfferedPriceQuery.graphql"
+import { PartnerOfferedPrice_artwork$data } from "__generated__/PartnerOfferedPrice_artwork.graphql"
+import { Details_artwork$data } from "__generated__/Details_artwork.graphql"
+
+interface PartnerOfferedPriceProps {
+  artwork: PartnerOfferedPrice_artwork$data
+  fallback: object
+}
+
+const PartnerOfferedPrice: React.FC<PartnerOfferedPriceProps> = ({
```

Comment:
This is the artwork price line which takes the server-side rendered prices (publicly listed price) as the placeholder and dynamically overwrite it, if there is a partner offer fetched on the client side.

I think I'll need to rename this component to make it clear.